airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group.json

```diff
@@ -2,108 +2,219 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
+    "customer.id": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
     "campaign.id": {
-      "type": ["null", "integer"]
```

Comment:
Could you please revert to old formatting? 
We are using it everywhere in the code.